executor/transaction_processor.go

```diff
@@ -121,9 +121,14 @@ type ethTestProcessor struct {
 // Process transaction inside state into given LIVE StateDb
 func (p *ethTestProcessor) Process(state State[txcontext.TxContext], ctx *Context) error {
 	// This check needs to be done before ApplyMessage is called, otherwise different state-root hash is produced
-	if len(state.Data.GetMessage().BlobHashes)*params.BlobTxBlobGasPerBlob > params.MaxBlobGasPerBlock {
-		return errors.New("blob gas exceeds maximum")
+	msg := state.Data.GetMessage()
+	if len(msg.BlobHashes)*params.BlobTxBlobGasPerBlob > params.MaxBlobGasPerBlock {
+		txHash := common.HexToHash(fmt.Sprintf("0x%016d%016d", state.Block, state.Transaction))
+		blockHash := common.HexToHash(fmt.Sprintf("0x%016d", state.Transaction))
+		ctx.ExecutionResult = newTransactionResult(ctx.State.GetLogs(txHash, uint64(state.Block), blockHash), msg, nil, errors.New("blob gas exceeds maximum"), msg.From)
```

Comment:
Since `GetLogs` is known to return an empty list in this case -- since no transaction was executed -- would it be fine to just pass an empty list to `newTransactionResult` instead of asking the state for an empty list? Something like
```suggestion
		ctx.ExecutionResult = newTransactionResult([]*types.Log{}, msg, nil, errors.New("blob gas exceeds maximum"), msg.From)
```
or maybe even
```suggestion
		ctx.ExecutionResult = newTransactionResult(nil, msg, nil, errors.New("blob gas exceeds maximum"), msg.From)
```

Reply:
Is this still applicable even with this PR? https://github.com/Fantom-foundation/Aida/pull/1201

Reply:
Now that im thinking about it more, you are right. It makes no sense to actually call the logs here.